go/libkb/skb_keyring.go

```diff
@@ -82,8 +82,15 @@ func (k *SKBKeyringFile) loadLocked() (err error) {
 			k.G().Log.Debug("| Keybase secret keyring doesn't exist: %s", k.filename)
 		} else {
 			k.G().Log.Warning("Error opening %s: %s", k.filename, err)
+			if os.IsPermission(err) {
```

Comment:
can you factor this out with the config.json file panic?

Reply:
sure.